apps/notifications/src/panel/templates/shortcuts-popover.jsx

```diff
@@ -3,7 +3,7 @@ import clsx from 'clsx';
 import { useTranslate } from 'i18n-calypso';
 import { useRef } from 'react';
 import { connect } from 'react-redux';
-import userAgent from 'calypso/lib/user-agent';
+import { isMobile } from 'calypso/lib/user-agent';
```

Comment:
Note this will require separate testing and deploying of the notifications app. 

See PCYsg-elI-p2 and PCYsg-OT6-p2

Reply:
> Note this will require separate testing and deploying of the notifications app.

It's not entirely clear to me from those resources: Is it on me to see through the deployment of that application with these changes, or will it be included in some regular future deployment?

Reply:
Yep, if you merge a change that touches a specific app, you should also smoke test and deploy that app. That's something that's often overlooked, as we don't currently have a mechanism to nudge or force people to do it.